auth-api/src/main/java/io/stargate/auth/api/AuthResource.java

```diff
@@ -32,9 +33,12 @@
 
 @Path("/v1")
 public class AuthResource {
+
   private static final Logger logger = LoggerFactory.getLogger(AuthResource.class);
 
   final AuthenticationService authService;
+  private static final boolean shouldEnableUsernameToken =
+      Boolean.parseBoolean(System.getProperty("stargate.auth_api_enable_username_token", "false"));
```

Comment:
where this property is set? I see it only set in the test:
https://github.com/stargate/stargate/pull/111/files#diff-2c65e0c4892da78e91b8b4d3d357e25aR406
but not in the actual `Starter` logic.

Reply:
Right. It's just a system property